src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -620,6 +620,7 @@ public boolean hasNoOngoingBusinessProcess() {
     private JudgmentInstalmentDetails joJudgmentInstalmentDetails;
     private LocalDate joPaymentToBeMadeByDate;
     private YesOrNo joIsLiveJudgmentExists;
+    private LocalDate joSetAsideDate;
```

Comment:
Do we need to save this confirmation joConfirmReadOrderForSetAside in case data?